src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,51 @@
+.nav {
+  font-weight: 500;
+  font-size: 12px;
+
+  &__list {
+    list-style-type: none;
+    padding: 0;
+    margin: 0;
+    display: flex;
+
+    &-item {
+      margin-left: 20px;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), for margin and padding use multiples of 4px. Consider changing `margin-left: 20px;` to `margin-left: 16px;` or `margin-left: 24px;`.